src/Ilios/CoreBundle/Service/Config.php

```diff
@@ -61,7 +61,11 @@ protected function getValueFromEnv($name)
     {
         $envName = 'ILIOS_' .  s($name)->underscored()->toUpperCase();
         if (isset($_SERVER[$envName])) {
-            return $_SERVER[$envName];
+            $result = $_SERVER[$envName];
+            if (in_array($result, ['null', 'false', 'true'])) {
```

Comment:
may wanna deal with capitalized/uppercased variants as well. 
suggesting a `strtolower()` call around `$result` on this line.

Reply:
Thanks - exposed a whole other issue with `json_decode` that it only handles the lowercase string.